### PR TITLE
SDK-203 [Java SDK] SDK should return create/update dates on read/find

### PR DESCRIPTION
**Pull request by VladimirLevitskiy**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-203](https://incountry.atlassian.net/browse/SDK-203)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

**Review by robert-par — APPROVED** [review]

**Review by robert-par — DISMISSED** [review]

File: src/main/java/com/incountry/residence/sdk/Storage.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
      * Write data to remote storage
      *
      * @param country country identifier
-     * @param record  object which encapsulate data which must be written in storage
+     * @param record  object which encapsulate data which must be written in storage*
```
Comment (RommeroInCountry) [inline src/main/java/com/incountry/residence/sdk/Storage.java:22]: typo?

**Review by RommeroInCountry — DISMISSED** [review]
LGTM

File: src/main/java/com/incountry/residence/sdk/Storage.java (hunk 1 above)
Comment (VladimirLevitskiy) [reply src/main/java/com/incountry/residence/sdk/Storage.java]: fixed

**Review by RommeroInCountry — APPROVED** [review]
LGTM

**Review by robert-par — APPROVED** [review]

**Pull request merged**